packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -471,15 +474,43 @@ export function generateGrid(
         const intl = useIntl();
         const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("${gqlTypePlural}Grid") };
         ${hasScope ? `const { scope } = useContentScope();` : ""}
+        
 
         const columns: GridColDef<GQL${fragmentName}Fragment>[] = [
             ${gridColumnFields
                 .map((column) => {
+                    const defaultMinWidth = 150;
```

Comment:
Shouldn't this have the same logic as below? Like checking for minWidth and maxWidth etc.

Reply:
FYI: The logic to set the column width is in lines 488-501. It incorporates the logic from before and sets `tooltipColumnWidth` and prepares the values to set them in lines 541 to 543.